articles/libraries/lock/v10/authentication-modes.md

```diff
@@ -2,23 +2,23 @@
 section: libraries
 description: Details about Popup Mode with Lock V10.
 ---
-# Popup Mode
+# Authentication Modes
```

Comment:
Makes sense. I really like the idea of breadcrumbs in the docs. I know a lot of people don't like them (heck, I don't really like them) but it could be really valuable in sections where the sidebar is not exhaustive or the docs didn't merit being included in the sidebar.